jerry-core/debugger/jerry-debugger-ws.c

```diff
@@ -13,8 +13,6 @@
  * limitations under the License.
  */
 
-#ifdef JERRY_DEBUGGER
```

Comment:
Fair point. It's strange that no testing caught the issue (or perhaps not that strange, because we have CI for Linux and OSX only). I've moved the platform-dependent includes within the guard.